src/gitattributes.ts

```diff
@@ -13,6 +13,8 @@ export class GitAttributesFile extends FileBase {
     super(scope, ".gitattributes", {
       editGitignore: false,
     });
+
+    this.project.addPackageIgnore("/.gitattributes");
```

Comment:
Actually never mind the race condition. You just have to add this in the pre-synth phase:
```suggestion
}
  public preSynthesize(): void {
    this.project.addPackageIgnore("/.gitattributes");
```